window_manager/src/main.cpp

```diff
@@ -0,0 +1,188 @@
+#define MY_IMPL
+
+#ifdef MY_IMPL
+#include <rcbe-engine/core/WindowManager.hpp>
+#include <rcbe-engine/utils/output_utils.hpp>
+#include <rcbe-engine/datamodel/system/window_config.hpp>
+#include <rcbe-engine/core/AbstractInputManager.hpp>
+#include <rcbe-engine/core/EditorInputManager.hpp>
+#include <rcbe-engine/datamodel/visual/Texture.hpp>
+
+#include <boost/log/trivial.hpp>
+
+int main(int argc, char* argv[]) {
+    try {
+        rcbe::utils::setup_logging(boost::log::trivial::debug);
```

Comment:
Even though this is sort of a debug tool in the first place, debug output should be configurable, so this should be removed.